event_operating_unit/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
```

Comment:
Is this module LGPL or AGPL ? Please unify :wink:

Reply:
go LGPL please